src/compile/data/assemble.ts

```diff
@@ -307,8 +312,8 @@ export function assembleFacetData(root: FacetNode): VgData[] {
  * @param  data array
  * @return modified data array
  */
-export function assembleData(dataCompomponent: DataComponent): VgData[] {
-  let roots: SourceNode[] = vals(dataCompomponent.sources);
+export function assembleData(model: Model, dataComponent: DataComponent): VgData[] {
```

Comment:
Unless, of course, there's some other way I can get access to the model's selection components down this code path!